docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2138,3 +2138,50 @@ Result:
 
 -   [tcp_port](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-tcp_port)
 
+## currentRoles {#current-roles}
+
+Returns the names of the roles which are current for the current user. The command [SET ROLE](../../sql-reference/statements/set-role.md#set-role-statement) could be used to change the current roles, by default the current roles are returned by the function [defaultRoles](#default-roles) called when the current user logins.
+
+**Syntax**
+
+``` sql
+currentRoles()
+```
+
+**Returned value**
+
+-   List of the current roles for the current user. 
+
+Type: [Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md)).
+
+## enabledRoles {#enabled-roles}
+
+Returns the names of the current roles merged with those roles which are granted to some of the current roles.
+
+**Syntax**
+
+``` sql
+enabledRoles()
+```
+
+**Returned value**
+
+-   List of the enabled roles for the current user. 
+
+Type: [Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md)).
+
+## defaultRoles {#default-roles}
+
+Returns the names of the roles which are set as current when the user logins. The command [SET DEFAULT ROLE](../../sql-reference/statements/set-role.md#set-default-role-statement) could be used to change the default roles. By default are all the roles which are granted to a user.
```

Comment:
By default the default roles are all the roles which are granted to a user

Reply:
Returns the names of the roles which are set as current when the current user logins.